include/deal.II/distributed/shared_tria.h

```diff
@@ -151,6 +170,17 @@ namespace parallel
        */
       std::vector<types::subdomain_id> true_subdomain_ids_of_cells;
     };
+
+    template <int dim, int spacedim>
+    template <class Archive>
+    void
+    Triangulation<dim,spacedim>::load (Archive &ar,
+                                       const unsigned int version)
+    {
+      dealii::Triangulation<dim,spacedim>::load (ar, version);
+      partition();
```

Comment:
Is this even necessary? Doesn't serializing the base class already stored the `subdomain_id` flags?